README.md

```diff
@@ -2,6 +2,26 @@ Disruptor
 ---
 [![built with nix](https://builtwithnix.org/badge.svg)](https://builtwithnix.org)
 
+### Example
+```bash
+$ ./nix-shell.sh
+...
+$ cd example_project_bzl_4
+$ bazel build //...
+```
+
+### Development shell
+Simply type run `./nix-shell.sh` to be dropped into a shell with all batteries included. 
+
+Project utilizes [nix](https://builtwithnix.org) to deliver a uniform, homogeneous development environment for each of its users & contributors - you are going to be using the same dependencies (bash, core utils, system libraries, bazel etc.), installed from the same source and configured in the same way. No more “works on my machine”-syndrome, as everyone has reproducibility identical working environment. 
```

Comment:
"reproducible and identical working environments" rather than "reproducibility identical working environment".

Reply:
Fixed, thanks!